tests/sql/internal.slt

```diff
@@ -1,108 +1,7 @@
 query I
-select * from pg_catalog.contributors;
+select count(github_id) > 50 from pg_catalog.contributors;
```

Comment:
note that GitHub as request limits for such things, maybe we need to exclude it from CI?

Reply:
Hmmm, looks like it makes more trouble than hardcoding. Let me revert that.